Lesson23/Mesut/filter.js

```diff
@@ -269,12 +281,19 @@ const productsExercise6 = [
 	},
 ];
 
+const productsExercise6Result= productsExercise6.filter(product=>product.quantity===0);
```

Comment:
this should be the othe way around, now you are only leaving products that are out of stock.
```suggestion
const productsExercise6Result= productsExercise6.filter(product=>!product.quantity===0);
```

or 
```suggestion
const productsExercise6Result= productsExercise6.filter(product=>product.quantity > 0);
```

Reply:
oh , I have misunderstood the term " filter out"